controllers/rootpolicystatus/root_policy_status_controller.go

```diff
@@ -91,53 +104,8 @@ func (r *RootPolicyStatusReconciler) Reconcile(ctx context.Context, request ctrl
 
 	log.Info("Updating the root policy status")
```

Comment:
Can you move this log to inside `RootStatusUpdate` since we don't know if the status will actually get updated?